tests/jsx_spread/jsfmt.spec.js

```diff
@@ -1 +1 @@
-run_spec(__dirname, ["babylon"]);
+run_spec(__dirname, ["babylon", "flow"]);
```

Comment:
@j-f1 there's already a test in the codebase: jsx_spread. I enabled it for the flow parser